ci/prow-entrypoint.sh

```diff
@@ -123,12 +123,7 @@ kola_test_qemu() {
 
     # Skip Secure Boot tests on SCOS for now
     # See: https://github.com/openshift/os/issues/1237
-    if [[ -f "src/config.json" ]]; then
-        variant="$(jq --raw-output '."coreos-assembler.config-variant"' 'src/config.json')"
-    else
-        variant="default"
-    fi
-    if [[ "${variant}" != "scos" ]]; then
+    if rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
```

Comment:
This logic looks inverted. If `centos-release` is present then we need to skip Secure Boot tests, not the other way around.

I think the reason this passed CI is that this condition happened to fail on another error, which is that `manifest` is not defined:

```
/src/ci/prow-entrypoint.sh: line 126: manifest: unbound variable
```

And I think the reason `set -e` didn't kick in here is likely a combination of it being in an if-statement and piping things.

Reply:
Proposing to fix this in https://github.com/openshift/os/pull/1458.

Reply:
Arg, thanks for noticing